tests/test_settings_api.py

```diff
@@ -9,8 +9,11 @@
 def test_setup_models_viscous_model_settings(new_solver_session) -> None:
     solver_session = new_solver_session
     case_path = download_file("elbow_source_terms.cas.h5", "pyfluent/mixing_elbow")
-    solver_session.file.read_case(file_name=case_path)
-    solver_session.solution.initialization.hybrid_initialize()
```

Comment:
The test seems to be able to set up the viscous model normally without initializing the case. Is there a reason the original test initialized the case?

Reply:
Not that I know of.